auth/realm/base/src/main/java/org/wildfly/security/auth/realm/LegacyPropertiesSecurityRealm.java

```diff
@@ -470,6 +472,7 @@ public Builder setDefaultRealm(String defaultRealm) {
          * @return this {@link Builder}
          */
         public Builder setPlainText(boolean plainText) {
+            Assert.checkNotNullParam("plainText", plainText);
```

Comment:
Since this attribute is of type `boolean` we do not have to check whether it is null. This will be done by the compiler

Reply:
Makes sense, removed the redundant check